src/components/BalanceForm.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Button, TextField, Typography, } from '@mui/material';
+import { AbstractProvider, formatEther, isAddress } from 'ethers';
+import { FormEvent, useState } from 'react';
+
+export default function BalanceForm() {
+  const ethers = require('ethers');
+  const provider: AbstractProvider = ethers.getDefaultProvider();
+
+  const [address, setAdress] = useState<string>('');
+  const [balance, setBalance] = useState<string>('');
+  const [balanceError, setBalanceError] = useState<string>('');
+
+  const getBalance = (address: string) => {
+    provider.getBalance(address)
+      .then((balance: bigint) => {
+        setBalance(formatEther(balance));
+        setBalanceError('');
+      })
+      .catch(() => setBalanceError('Something goes wrong 😢'))
+  }
+
+  const handleSubmit = (e:FormEvent<HTMLFormElement>): void => {
+    e.preventDefault();
+    
+    if(!isAddress(address)) {
+      setBalanceError('Invalid ETH address');
+    }else if (address === '') {
+      setBalanceError('Address is requaried');
+    }else {
+      getBalance(address);
+    }
```

Comment:
you shoud donwload Prettier extension and always format your code
it won't affect functionality but code will be more readable

```
if (!isAddress(address)) {
  setBalanceError('Invalid ETH address');
} else if (address === '') {
  setBalanceError('Address is requaried');
} else {
  getBalance(address);
}
```